config/routes.rb

```diff
@@ -118,9 +118,6 @@
   # advanced routes for advanced search
   get 'search' => 'advanced#index', as: :advanced
 
-  # Authority vocabulary queries route
```

Comment:
@jcoyne I'm surprised there is no route for searching the Authorities needed.  Is this coming in from Questioning Authority?